recipes/CompareText2Img.py

```diff
@@ -274,7 +278,16 @@ def get_raw_price(self, state: dict) -> int:
                 case Text2ImgModels.dall_e.name | Text2ImgModels.dall_e_3.name:
                     total += 15
                 case Text2ImgModels.gpt_image_1.name:
-                    total += 20
+                    if cost := grouped_costs.get(Text2ImgModels.gpt_image_1.name):
+                        total += math.ceil(float(cost) * 2.5)
+                        continue
+                    match state.get("gpt_image_1_quality"):
+                        case "low":
+                            total += 3
+                        case "medium":
+                            total += 10
+                        case _:
+                            total += 40
```

Comment:
not sure if its worth keeping this maths around? given that usage costs are tracked. just do a default of 20

Reply:
this is to show a somewhat accurate estimate before the run